trunk/source/Library-Automata/build.properties

```diff
@@ -2,6 +2,5 @@ source.. = src/
 output.. = bin/
 bin.includes = .,\
                META-INF/,\
-               src/de/uni_freiburg/informatik/ultimate/automata/petrinet/petruchio/petruchio.jar,\
-               src/de/uni_freiburg/informatik/ultimate/automata/nestedword/operations/optncsb/trove-3.0.3.jar
+               src/de/uni_freiburg/informatik/ultimate/automata/petrinet/petruchio/petruchio.jar
```

Comment:
What about petruchio.jar? Is this not available from Maven? Would this be a candidate for packaging in our p2 repo?

Reply:
Yes, it is not available on Maven (it is pretty old). But if we can use our P2 repo for such JARs, I am absolutely in favour of this.

Reply:
In that case, we can also store the Eldarica JAR on our P2 (or figure out what depencies it actually contains).

Reply:
But this can be a separate PR; it just would be nice to get rid of these "hidden" jars.